### ISSUE TITLE
Add support for uploading poster images

### ISSUE THREAD

**Issue by stevecassidy** [Signbank/signbank-video#1]
@henrinie said: 
We don't generate these on the server, instead our users pick a frame and upload it.
I think we would need a FileField for poster images.
The extract_frame part would need to be made optional (add a boolean to settings?)

**Comment by stevecassidy** [comment]
This would seem to be an easy change. I'd suggest that this package just implements this as an optional way to set the poster and provides forms for upload etc.   The way you use it can be configured in the parent project.   

When you say they pick a frame - would it be good to have the option of just sending the frame offset rather than having to extract the frame themselves? We could then extract the frame at that offset and store it.

**Comment by henrinie** [comment]
> I'd suggest that this package just implements this as an optional way to set the poster and provides forms for upload etc. The way you use it can be configured in the parent project.

I agree.

> ... would it be good to have the option of just sending the frame offset rather than having to extract the frame themselves?

It might be a good idea. However it doesn't change much for the user at the moment:
A user can select a desired position in the video, and press a button to upload it. If someone gets around implementing this the way you described it, I think it would be a good idea.

**Comment by stevecassidy** [comment]
Do you have this implemented in your project? Perhaps you could port it here - sounds like a facility that would be generally useful.

**Comment by henrinie** [comment]
Yes we have. I think I can port it in the project, but then we need to have that FileField (I did not use ImageField for this).
The feature simply takes a screenshot of the video at the selected point and the user then uploads the image.

I am working on another feature related to videos: recording videos with webcam. We would like to port it to signbank-video too. The drawback of it is that it will not work with all browsers, which is okay for us, is that okay with you?
The current situation with browser support can be seen here: https://github.com/Signbank/FinSL-signbank/wiki/Compatibility
